web/js/map-OpenLayers.js

```diff
@@ -513,7 +513,37 @@ $.extend(fixmystreet.utils, {
         }
     }
 
-    function marker_click(problem_id, evt) {
+    var defect_popup;
+
+    function close_defect_popup() {
+        if (!!defect_popup) {
+            fixmystreet.map.removePopup(defect_popup);
+            defect_popup.destroy();
+            defect_popup = null;
+        }
+    }
+
+    function marker_click(feature, evt) {
+        close_defect_popup();
+
+        var problem_id = feature.attributes.id;
+
+        // Oxfordshire: Handle clicks on defects, which have negative fake IDs.
+        if (problem_id < 0) {
+            defect_popup = new OpenLayers.Popup.FramedCloud(
+                "occDefects",
+                feature.geometry.getBounds().getCenterLonLat(),
+                null,
+                feature.attributes.title.replace("\n", "<br />"),
+                { size: new OpenLayers.Size(0, 0), offset: new OpenLayers.Pixel(0, 0) },
```

Comment:
The overlapping of the popover and the marker is a bit odd to my eye:

<img width="335" alt="image" src="https://user-images.githubusercontent.com/4776/97337311-07be8280-1878-11eb-995b-42464061c99a.png">

If this was instead `{ size: new OpenLayers.Size(0, 0), offset: new OpenLayers.Pixel(6, -46) },` then the bubble could almost be imagined to be coming from the person in the icon:

<img width="345" alt="image" src="https://user-images.githubusercontent.com/4776/97337369-19078f00-1878-11eb-960c-d090107172f2.png">

Reply:
Love the idea of it being a speech bubble!

Fixed in d170b80c0865b2c67e3f51d0647afa2283cc7498 and 0d45ab49e3ea6480340e9c839d70f842d2324d56.